### PR TITLE
Polish groupedItem button & add groupedItem to the existing browserButtons

### LINKED ISSUE #9251: buttons for commonForm on about:styles need updating
**Describe the issue you encountered:** buttons inside `<pre>` for `commonForm` on `about:styles` need updating.

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### LINKED ISSUE #9252: Polish groupedItem button
**Describe the issue you encountered:** Polish groupedItem button.

Remove margin from `browserButton` and add it to `groupedItem`. A button as such does not require margin. Since the button requires the margin when it is grouped with another button in the same wrapper, adding margin to `BrowserButton` with `groupedItem` is a better solution than adding it to all buttons, some of whose margin would be canceled with `margin: 0`.

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes #9251
Fixes #9252

Test Plan:
1. Open about:styles
2. Click `buttons`
3. Make sure the buttons with 'groupedItem' have margin-left only

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/renderer/components/common/browserButton.js [hunk 1]
```diff
@@ -188,8 +192,19 @@ const styles = StyleSheet.create({
     //
     // TODO: remove !important and check advancedSettings.js
     // once preferences is fully refactored
-    marginRight: '4px !important',
-    marginLeft: '4px !important'
+
+    // 'mm' assures theoretically the equal width of margin
+    // on every platform with any display monitor resolution.
+    // 2 mm = 1/12 inch
+    marginLeft: '2mm !important',
```
Comment (luixxiul) [inline app/renderer/components/common/browserButton.js]: the value might be changed to a better one.

Comment (cezaraugusto) [inline app/renderer/components/common/browserButton.js]: I have a strong guess that most monitors aren't yet in the same high-res as printers to get absolute units with the same precision. Absolute units are basically made for printed pages.

If you're interested, please check https://www.w3.org/Style/Examples/007/units.en.html and lmk if that changed your mind or, if not, lmk your thoughts.

Comment (cezaraugusto) [inline app/renderer/components/common/browserButton.js]: otherwise lgtm

Comment (luixxiul) [inline app/renderer/components/common/browserButton.js]: > Absolute units are basically made for printed pages.

yes I think you are right. I'll take a look at the document later.

Comment (luixxiul) [inline app/renderer/components/common/browserButton.js]: Fixed with https://github.com/brave/browser-laptop/commit/b6491058510ae3f0ae723b815617ab25ea61f926

**Comment by luixxiul** [comment]
The comment was addressed and the commits were updated :-)

**Pull request merged**